### ISSUE TITLE
rename unix/linux/core.c

### ISSUE THREAD

**Issue by philips** [joyent/libuv#464]
Gyp now throws an error if two files have the same basename. You can see the context here:

 https://groups.google.com/forum/?fromgroups#!topic/gyp-developer/QN-nYWTMUkM

This is the only file in the tree with this problem so rename it to linux-core.c or something. 

IRC context:

```
[10:44:32] <philips> piscisaureus_: The gyp guys responded with a reasonable response
[10:44:41] <piscisaureus_> philips: link?
[10:44:41] <philips> piscisaureus_: It turns out libtool warns about this too
[10:44:46] <piscisaureus_> oh
[10:45:06] <piscisaureus_> I think bnoordhuis should fix it, then :-)
[10:45:14] <piscisaureus_> rename linux/core.c
[10:45:15] <philips> piscisaureus_: https://groups.google.com/forum/?fromgroups#!topic/gyp-developer/QN-nYWTMUkM
[10:45:32] <philips> piscisaureus_: Yea, I can send a patch if bnoordhuis can't do it.
[10:46:01] <piscisaureus_> he can :-)
[10:46:18] <philips> piscisaureus_: heh, sweet ;)
[10:46:43] <philips> http://www.jankratochvil.net/project/libtool/
```

After fixing this we can use gyp as a submodule: `http://git.chromium.org/external/gyp.git`

**Comment by bnoordhuis** [comment]
Fixed in e54cda5.

**Comment by bnoordhuis** [comment]
Make that e132075.
